onboard/catkin_ws/src/cv/scripts/depthai_simulate_detection.py

```diff
@@ -213,16 +216,11 @@ def _run_detection_on_image_topic(self, device):
         """
         TopicType, _, _ = rostopic.get_topic_class(self.feed_path)
         rospy.Subscriber(self.feed_path, TopicType, self._update_latest_img)
-        loop_rate = rospy.Rate(1)
```

Comment:
Is there a particular reason for not limiting the loop frequency?

Reply:
Back when we developed the first DepthAI script, we needed to limit the loop frequency for it to work properly. However, later versions of DepthAI scripts, including the current versions, do not require a limit on the loop frequency to function. In fact, we discovered the limit on the loop frequency as significantly reducing the detection rate, which we needed to improve. Therefore, we got rid of the limits on the loop frequency.